bitnami/rabbitmq-cluster-operator/values.yaml

```
@@ -131,6 +131,9 @@ clusterOperator:
     ##   - myRegistryKeySecretName
     ##
     pullSecrets: []
+  ## @param clusterOperator.revisionHistoryLimit sets number of replicaset to keep in k8s
+  ##
+  revisionHistoryLimit: 3
```

What if we honor the Kubernetes default value which is `10`?
```suggestion
  revisionHistoryLimit: 10
```